extensions/omni/hook_harness.c

```diff
@@ -148,6 +160,14 @@ MODULE_FUNCTION void omni_check_password_hook(const char *username, const char *
                 validuntil_null);
 }
 
+MODULE_FUNCTION PlannedStmt *omni_planner_hook(Query *parse, const char *query_string,
+                                               int cursorOptions, ParamListInfo boundParams) {
+  PlannedStmt *stmt =
+      iterate_hooks(omni_hook_planner, parse, query_string, cursorOptions, boundParams)
```

Comment:
This has to be changed to `iterate_hooks(planner` (similarly to all adjacent hooks) to account for changes introduced in omni's inner workings (see 4ced20bec69b98cab68e23abc4d3b42930dffe39)